code/modules/mapping/mapping_helpers.dm

```diff
@@ -165,6 +165,223 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/effect/mapping_helpers)
 	else
 		airlock.abandoned = TRUE
 
+/obj/effect/mapping_helpers/airlock/access
```

Comment:
This should override existing access on airlocks and wipe them completely.

Reply:
If you have multiple access helpers on top of one another, it will wipe the previous one too.

For example in the gravity generator on Box.

![image](https://github.com/user-attachments/assets/393a5b9b-49bd-4745-8c33-4683832b753d)

Reply:
True